src/commands/trialApp/removeTrialApp.ts

```diff
@@ -13,7 +14,13 @@ export async function removeTrialApp(context: IActionContext, node?: TrialAppTre
         node = await ext.tree.showTreeItemPicker<TrialAppTreeItem>(TrialAppTreeItem.contextValue, context);
     }
 
-    ext.context.globalState.update(TrialAppLoginSession, undefined);
-    delete ext.azureAccountTreeItem.trialAppNode;
-    await ext.tree.refresh();
+    const message: string = `Are you sure you want to remove Trial app "${node.client.fullName}"?`;
+    const remove: MessageItem = { title: 'Remove' };
+    const response: MessageItem | undefined = await ext.ui.showWarningMessage(message, { modal: true }, remove);
+
+    if (response === remove) {
```

Comment:
I think you technically don't have to this check here since if the user hits cancel, it'll throw a UserCancelledError  (since there is only one choice here, if it passes, we can assume the response we remove).

Means we don't need that const response either.